File: baekjoon/1차원 배열/2577.py
A = int(input())
B = int(input())
C = int(input())
ABC = A * B * C
result = [0,0,0,0,0,0,0,0,0,0]
while ABC != 0:
  ABC,nmg = divmod(ABC,10)
  result[nmg] = result[nmg] + 1
for i in result:
  print(i)

File: baekjoon/단계별로 풀어보기/1차원 배열/2577.js
const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

let input = []
const handleInput = (line) => {
  input.push(Number(line))
}

const handleOutput = () => {
  const [A, B, C] = input
  const numStr = (A * B * C).toString()
  let result = new Array(10).fill(0)
  for (let i = 0; i < numStr.length; i++) {
    result[Number(numStr[i])]++
  }
  for (i in result) {
    console.log(result[i])
  }
}

rl.on('line', handleInput).on('close', handleOutput)
